ext/RMagick/rmmain.c

```diff
@@ -14,6 +14,13 @@
 #define MAIN                        // Define external variables
 #include "rmagick.h"
 
+#if defined HAVE_SETMAGICKALIGNEDMEMORYMETHODS
+#if defined HAVE_RB_GC_ADJUST_MEMORY_USAGE && \
+    (defined HAVE_POSIX_MEMALIGN || defined HAVE_MEMALIGN || defined _WIN32)
```

Comment:
@Watson1978 what platforms do and don't have these options? Is it just Ruby 2.3 that doesn't have them, or are there others? Will we be able to remove these checks at some point?

Reply:
The platforms that posix API cannot be used ￼and they doesn’t have ImageMagick 7.0.9

Reply:
I have tried with macOS, Linux and Windows 10. So, almost platforms will be supported, I think.